pkg/fixer/fixes/fmt.go

```diff
@@ -0,0 +1,55 @@
+package fixes
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+
+	"github.com/open-policy-agent/opa/format"
+)
+
+type Fmt struct {
+	// KeyOverride allows this fix config to also be registered under another key, see note
+	// in Key().
+	KeyOverride string
+	// OPAFmtOpts are the options to pass to OPA's format.SourceWithOpts
+	// function.
+	OPAFmtOpts format.Opts
+}
+
+func (f *Fmt) Key() string {
+	// this allows this fix config to also be registered under another key so that different
+	// configurations can be registered under other linter rule keys.
+	if f.KeyOverride != "" {
+		return f.KeyOverride
+	}
+
+	return "opa-fmt"
+}
+
+func (*Fmt) WholeFile() bool {
```

Comment:
Do we need this any longer? From what I can tell these are never called.

Reply:
Ahh, oops. Fixed in f05782e1cc5cc1416ed6672da5d298baa9ec9353